src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java

```diff
@@ -171,14 +171,6 @@ public RightClickMenu(JabRefFrame frame, BasePanel panel) {
             }
         });
 
-        add(new GeneralAction(Actions.MERGE_DOI, Localization.lang("Get BibTeX data from DOI")) {
```

Comment:
Next step would be to add https://github.com/JabRef/jabref/issues/1098, so I prefer not to add it back in this context menu.